ports/espressif/boards/lolin_c3_mini/mpconfigboard.mk

```diff
@@ -1,7 +1,5 @@
-USB_VID = 0x303A
-USB_PID = 0x80F5
-USB_PRODUCT = "Lolin C3 Mini"
-USB_MANUFACTURER = "Wemos"
+CIRCUITPY_CREATOR_ID =  0x1988_1988
```

Comment:
It appears that this is causing the build to fail, as it is expecting the CIRCUITPY_CREATOR_ID to be an integer. [Based on another example of somebody using a creator ID](https://github.com/adafruit/circuitpython/blob/14b734be6ed7d8a2644610d0fd899f516deae51d/ports/espressif/boards/lilygo_ttgo_t-01c3/mpconfigboard.mk#L1), it seems you could fix this problem simply by removing the underscore

Reply:
Thanks @kylefmohr  !